.vsts-ci.yml

```diff
@@ -0,0 +1,56 @@
+phases:
+- template: /eng/build.yml
+  parameters:
+    agentOs: Windows_NT
+    queue:
+      ${{ if in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:
+        name: Hosted VS2017
+      ${{ if notIn(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:        
+        name: DotNetCore-Build
+        demands:
+        - agent.os -equals Windows_NT        
+      parallel: 2
+      matrix:
+        Build_Debug:
+          _BuildConfig: Debug
+          _PublishType: none
+          _SignType: test
+        Build_Release:
+          _BuildConfig: Release
+          ${{ if in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:
```

Comment:
Did you intend to change the `Build.Reason` for all of this to just `PullRequest` in a separate PR when you do the official build changes?

Reply:
Yes. Subsequent PR. Once I have pull request validation happening, I will switch to making CI working.